components/Chips/tests/unit/ChipsTests.m

```diff
@@ -294,6 +294,39 @@ - (void)testPointInsideWithCustomHitAreaInsets {
                         withEvent:nil]);
 }
 
+- (void)testRemoveChipsManually {
+  // Given
+  MDCChipField *field = [[MDCChipField alloc] init];
+  field.frame = CGRectMake(0, 0, 200, 50);
+  field.textField.text = @"Test";
+  field.textField.placeholder = @"Test";
+  [field setNeedsLayout];
+  [field layoutIfNeeded];
+  CGFloat initialPlaceholderOriginX =
+      CGRectStandardize(field.textField.placeholderLabel.frame).origin.x;
+
+  // When
```

Comment:
Each test should have a single "When" and "Then" section. I believe that getting the position of the initial X position and the "with chip" X position are both part of the "When" or "Given" phases of the test.

Consider this suggestion:

```
// Given
Prepare chipfield
Add a chip, determine the location of placeholder

// When
Remove the chip

// Then
Get the position of the placeholder
Verify that the placeholder X-position is less than the initial value (with the Chip)
```